build/common.props

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<!--Semantic Version-->
+	<PropertyGroup>
+		<!-- 
+		  Semantic Version. See http://semver.org for full details. 
+		  Update for every public release. 
+		-->
+		<SemanticVersionMajor>3</SemanticVersionMajor>
+		<SemanticVersionMinor>2</SemanticVersionMinor>
+		<SemanticVersionPatch>0</SemanticVersionPatch>
+		
+		<!-- 
+		Date when Semantic Version was changed. 
+		Update for every public release.
+		-->
+		<SemanticVersionDate>2015-8-21</SemanticVersionDate>
+
+		<!-- 
+		Pre-release version is used to distinguish internally built NuGet packages.
+		Pre-release version = Minutes since semantic version was set, divided by 5 (to make it fit in a UInt16).
+		-->
+		<PreReleaseVersion>$([MSBuild]::Divide($([System.DateTime]::Now.Subtract($([System.DateTime]::Parse($(SemanticVersionDate)))).TotalMinutes), 5).ToString('F0'))</PreReleaseVersion>
```

Comment:
+1 for using UTC. Prerelease version will be a consistent across different dev machines